src/lib/net/SecureSocket.cpp

```diff
@@ -335,6 +340,11 @@ bool SecureSocket::loadCertificates(std::string &filename)
   return true;
 }
 
+static int cert_verify_ignore_callback(X509_STORE_CTX *, void *)
```

Comment:
Free functions are normally defined separately--form the class functions--at the top of the source file.

Also it should be `camelCase`.

```suggestion
static int verifyIgnoreCertCallback(X509_STORE_CTX *, void *)
```